scvi/autotune/_manager.py

```diff
@@ -0,0 +1,492 @@
+import inspect
+import logging
+import warnings
+from collections import OrderedDict
+from typing import Any, Callable, List, Optional, Tuple
+
+import rich
+
+try:
+    from ray import air, tune
+    from ray.tune.integration.pytorch_lightning import TuneReportCallback
+except ImportError:
+    pass
```

Comment:
should we put a better message here?

Reply:
Ray import error messages are handled by the `@dependencies` decorator right now, otherwise there would be a message every time scvi is imported. Should I still include it here though?

Reply:
no all good I missed that sorry